pybsn/__init__.py

```diff
@@ -214,11 +214,11 @@ def rpc(self, path, data):
         :param params: request parameters to attach
         :return: the deserialized RPC output (for most RPCs, a dict).
         """
-        response = self._request("POST", path, data=self._dump_if_present(data), rpc=True)
-        if response.status_code == requests.codes.no_content:
-            return None
-        else:
+        response = self._request("POST", path, data=self._dump_if_present(data), rpc=True, params=params)
+        try:
             return response.json()
+        except ValueError:
```

Comment:
Slight preference for the previous logic here

Reply:
Starting async RPC currently returns 202 with no body failing the json parsing part was the reason it needed this change.

I thought about adding 202 Accept case to the if-clause here, 
but looked up RFC 7231 for 202 Accepted, 
`with this  response ought to describe the request's current status`
not sure how strong `ought` is supposed to mean, MUST or SHOULD
but either way server can legitimately respond with body, but we needs to tolerate current response without it and ended up with this try and None on parse error.

Reply:
my main concern was silently 'swallowing' json decoding errors for all RPC calls for something that only happens in a very limited scope..